### DISCUSSION TITLE
My GitHub account was hacked and later suspended due to violation of Terms and Services

### DISCUSSION THREAD

**Discussion by Moonknight27** [community/community#119843] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am unable to login into my github account since yesterday (**16th April, 2024**), morning (**Around 8:00 AM Indian Standard Time**). As I was already logged in from my device, I got the following message...
**Access to your account has been suspended due to a violation of our Terms of Service** (Kindly refer to the attached snapshots of the same)

I am sure that my account was hacked (as I have faced similar attacks on other platforms within the past few days) and I am sure that any malpractice that was carried out from my account was definitely not in my knowledge.

Since then I have not been able to login to my account from my device. Moreover, I am also not able to open GitHub's support page ([support.github.com](**http://support.github.com/**)), as it gives **HTTP_403_Forbidden** error everytime. I have sent 2 mails to GitHub Support Mail regarding this, but I haven't yet received a response from them via mail. As there is no other apparent way for me to connect with them, I am writing this article hoping for a response from GitHub team.

This GitHub account is very crucial to my work and academics, and I might face serious repercussions if I don't get back this account within a short time. I would deeply appreciate if anyone from the GitHub can escalate the appeal and or do anything about it; my original account is **"github.com/Technight27"**.

![IMG-20240416-WA0006](https://github.com/community/community/assets/167226311/5475b435-a513-425d-adc2-5d0efc36e064)
![IMG-20240416-WA0010](https://github.com/community/community/assets/167226311/258ce6e5-0126-461a-8726-1b66cbf51f1e)

**Comment by atlansdev** [comment]
Hey!

Exactly the same thing happened to me, my account was used in some way in another country without even asking for two-factor validation. A kind of cookie theft.

This caused my account to be suspended, my projects are frozen and stopped on github and I don't get the proper attention from the support team, if anything they gave me feedback on my open ticket.

My account is: github.com/wagnercrosa

**Comment by ebndev** [comment]
Hi @Moonknight27,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our Support page. Try [this link](https://support.github.com/contact/account) that will take you directly to the account form if the main site isn't working for you.

This is a public forum, and the community is unable to assist with nor escalate your ticket. When it is received, it will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
